hw/ip_templates/ac_range_check/rtl/ac_range_check.sv.tpl

```diff
@@ -274,18 +274,18 @@ module ${module_instance_name}
   prim_count #(
     .Width(DenyCountWidth)
   ) u_deny_count (
-    .clk_i              ( clk_i              ),
-    .rst_ni             ( rst_ni             ),
-    .clr_i              ( clear_log          ),
-    .set_i              ( 1'b0               ),
-    .set_cnt_i          ( '0                 ),
-    .incr_en_i          ( deny_cnt_incr      ),
-    .decr_en_i          ( 1'b0               ),
-    .step_i             ( DenyCountWidth'(1) ),
-    .commit_i           ( 1'b1               ),
-    .cnt_o              ( deny_cnt           ),
-    .cnt_after_commit_o (                    ),
-    .err_o              ( deny_cnt_error     )
+    .clk_i              ( clk_i                          ),
+    .rst_ni             ( rst_ni                         ),
+    .clr_i              ( 1'b0                           ),
+    .set_i              ( clear_log                      ),
```

Comment:
This is a bit surprising to me. Doesn't the count now get set to `1` if we get a clear with no increment?